src/commands/dxb/profile/passwordpolicies/fix.ts

```diff
@@ -29,26 +27,48 @@ export default class PasswordPoliciesMerge extends SfdxCommand {
     protected orgname:string;
 
     public async run() {
-        let profilename = this.flags.profilename;
-        let sourcepath = this.flags.sourcepath;
-        this.orgname = this.org.getUsername();
-        if (profilename) {
-            try {
-                //do something
-            } catch (err) {
-                console.log(`Could not convert ${profilename}`);
+        const sourcepath = this.flags.sourcepath;
+        try {
+            // check if sourcepath exists and continue
+            fs.accessSync(sourcepath);
+            // retrieve all password policies from target org, these have a different timestamp appended to the file name than the source files
+            fs.mkdirSync('targetOrgPolicies/main/default', { recursive: true });
+            const targetOrgPolicies: { filePath: string, fullName: string }[] = this.getAllPasswordPolicies();
+
+            // get the file names for the source files from the source directory
+            let sourceFiles:string[] = fs.readdirSync(sourcepath);
+            if (sourceFiles.length === 0) {
+                throw new Error(`No source files were found in ${sourcepath}`);
+            }
+            for (const file of sourceFiles) {
+                // find the profile password policy from the target org that starts with the same profile name as the source file
+                const targetOrgPolicyToReplace:string = (
+                    targetOrgPolicies.find(targetOrgPolicy => targetOrgPolicy.fullName.startsWith(file.split('profilePasswordPolicy')[0])) ||
+                    { filePath: `targetOrgPolicies/profilePasswordPolicies/${file}` } // if profile policy does not exist in target, ensure it is copied over
+                ).filePath;
+
+                // copy the content of the source file to the target org profile password policy. This will only copy content and not the file name from the target
+                fs.copyFileSync(`${sourcepath}/${file}`, targetOrgPolicyToReplace);
+            }
+            // remove the source directory and it's content, recreate it after as an empty directory
+            fs.rmSync(`${sourcepath}`, { force: true, recursive: true });
```

Comment:
line 54 and 55 can be simplified in one line using fs.emptyDirSync(sourcePath)

Reply:
Ok, I will need to replace fs with fs-extra I believe.